its/tests/pom.xml

```diff
@@ -110,6 +116,35 @@
               <stripVersion>false</stripVersion>
             </configuration>
           </execution>
+          <execution>
+            <id>fetch-sloop-distribution</id>
+            <phase>generate-test-resources</phase>
+            <goals>
+              <goal>copy</goal>
+            </goals>
+            <configuration>
+              <artifactItems>
+                <artifactItem>
```

Comment:
Minor optimization, but we could use Maven profiles to only download the distribution of the current platform

Reply:
addressed in [0b76eac](https://github.com/SonarSource/sonarlint-core/pull/765/commits/0b76eacc86d4104f1638cad6914f45b521f5346a)